### PR TITLE
Harden storage lifecycle and simplify Hermes control

### PR DESCRIPTION
**Pull request by Th0rgal**
## Summary

- enforce aggressive, configurable workspace GC and disk-pressure admission guards
- add an operational hard-clean script and production-oriented storage defaults
- make Hermes chat the primary control surface, with diagnostics hidden in drawers by default
- surface Hermes HTTP/upstream failures, support CRLF SSE streams, and recover cleanly from aborted sessions
- add unit and browser coverage for the Hermes control flow

## Validation

- `cargo fmt --all --check`
- `cargo check`
- `cargo test mission_workspace_gc --lib`
- `cargo test disk_estimate_preserves_emergency_free_space --lib`
- `bash -n scripts/storage_hard_cleanup.sh`
- dashboard unit suite: 243 tests
- dashboard TypeScript check and targeted ESLint
- Hermes Playwright suite: 2 tests
- dashboard production build
- `git diff --check`

---

> [!NOTE]
> **Medium Risk**
> Mission creation and workspace deletion behavior change under new env defaults and admission rules; Hermes UI is a large UX shift but storage changes are the main operational risk on production hosts.
> 
> **Overview**
> **Disk and mission admission** get stricter, more configurable behavior: optional refusal of new missions at disk **warn** (`DISK_ADMISSION_AT_WARN`), a fail-safe default local disk estimate (`MISSION_DISK_DEFAULT_ESTIMATE_GIB`), and disk gates that apply only when `requires_local_disk` is true so **remote-node** missions skip host pressure checks. Workspace GC defaults shorten retention (1d terminal / 7d paused), honor `WORKSPACE_GC_INTERVAL_MINUTES`, run extra sweeps under disk pressure with a shared lock, and the disk watcher triggers those sweeps. A new **`storage_hard_cleanup.sh`** (dry-run/`--apply`) prunes host-level backups, logs, caches, and journal size; **`.env.example`** and **`STORAGE_LIFECYCLE.md`** document the aggressive build-host profile.
> 
> **Hermes dashboard** is reworked so **chat is the main surface**; mission control and mission updates live in **slide-over drawers** (Missions badge, updates bell, Escape to close). The thread gains quick-action chips, **retry** on failed sends, and clean reset when starting a new conversation mid-stream. **`hermes.ts`** improves HTTP errors (JSON `detail`), throws on failed stream setup, and **normalizes CRLF in SSE** parsing. Unit and Playwright tests cover pagination, layout, streaming abort, and API edge cases.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit 0f904d8d95228a383854cd3064abc4f9a390dc16. Bugbot is set up for automated code reviews on this repo. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

File: dashboard/src/components/hermes/alerts-feed.test.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    render(
+      <SWRConfig value={{ provider: () => new Map(), dedupingInterval: 0 }}>
+        <AlertsFeed />
+      </SWRConfig>,
+    );
+
+    fireEvent.click(await screen.findByRole("button", { name: "Load older" }));
+
+    await waitFor(() => {
+      expect(screen.queryByRole("button", { name: "Load older" })).not.toBeInTheDocument();
```
[the remark this answers is not in the document]
Comment (Th0rgal) [reply dashboard/src/components/hermes/alerts-feed.test.tsx:43]: Resolved by #658 (merge 6b8fd835): refreshed first-page alerts now reopen and deduplicate the pagination gap correctly.

File: dashboard/src/components/hermes/hermes-thread.tsx [hunk 2]
```diff
@@ -477,14 +506,28 @@ export function HermesThread({ className }: { className?: string }) {
...
+            className="rounded-lg border border-red-500/25 bg-red-500/10 px-3 py-2 text-xs text-red-300"
+          >
             {error}
           </p>
         )}
       </div>
 
       {/* Composer — same row grammar as the Ask panel composer */}
       <div className="border-t border-white/[0.06] px-6 py-3">
+        {error && input.trim() && !loading && (
```
[the remark this answers is not in the document]
Comment (Th0rgal) [reply dashboard/src/components/hermes/hermes-thread.tsx:520]: Resolved by #658 (merge 6b8fd835): stream failures preserve the draft and expose an explicit Retry action.
